CHANGELOG.md

```diff
@@ -2,21 +2,30 @@
 
 This changelog follows the following convention [https://keepachangelog.com/en/1.0.0/](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+## [1.2] - 2019-09-??
 
 ### Breaking
 
 - `textworld.envs.wrappers.Filter` expects the environment to wrap as its first argument.
 - `textworld.logic.State` now requires the `GameLogic` to be provided, so that it can know about the type hierarchy of each variable.
 
```

Comment:
Mention that `has_won` and `has_lost` were replaced with `won` and `lost`. I can (and should have) move those changes into a separate commit.